framework/source/class/qx/Class.js

```diff
@@ -1712,14 +1637,17 @@ qx.Bootstrap.define("qx.Class",
      *
      * @param base {Class} The base class.
      * @param mixins {Mixin[]} All mixins which should be included.
+     * @param type {String} the type of the class
      * @return {Boolean} true, if the constructor needs to be wrapped.
      */
-    __needsConstructorWrapper : function(base, mixins)
+    __needsConstructorWrapper : function(base, mixins, type)
```

Comment:
Since you removed the call to `__needsConstructorWrapper` above, is it necessary to even still have this method here?

Reply:
ah, no...